plugins/org.jkiss.dbeaver.model.sql/src/org/jkiss/dbeaver/model/sql/semantics/model/ddl/SQLQueryObjectDataModel.java

```diff
@@ -59,10 +57,10 @@ public class SQLQueryObjectDataModel extends SQLQueryRowsSourceModel implements
     private SQLQuerySymbolOrigin objectNameOrigin = null;
 
     public SQLQueryObjectDataModel(
-            @NotNull STMTreeNode syntaxNode,
-            @NotNull SQLQueryQualifiedName name,
-            @NotNull DBSObjectType objectTypes,
-            @NotNull Set<DBSObjectType> objectContainerTypes
+        @NotNull STMTreeNode syntaxNode,
+        @NotNull SQLQueryComplexName name,
```

Comment:
maybe use fullname for this field?

Reply:
It's the name of an object that might be represented as just a single identifier or as a fully qualified name. The common `name` is more suitable here